crates/cast/bin/cmd/run.rs

```diff
@@ -231,6 +232,36 @@ impl RunArgs {
         let result = {
             executor.set_trace_printer(self.trace_printer);
 
+            if let Some(signature) = tx.inner.signature {
+                let v = signature.v;
```

Comment:
not opposed to this, but I'd like to have this ideally as a separate function so we can properly document this and have this isolated.

this should probably also check the chainid?

Reply:
Hi @mattsse 

We agree on the first point to create a separate function, will update the PR

On the `chainId` topic:

Quorum explicitly doesn't allow a `chainId` of `1`, it will not run if it is configured this way
https://docs.goquorum.consensys.io/concepts/network-and-chain-id

Are you saying that if `chainId` is 1, then `is_private_quorum_txn` should be `false` here?

Reply:
@mattsse, I refactored this out into a separate function, not sure if you want it isolated via a CLI flag as well